### PR TITLE
Remove unnecessary `kCliAnimationsFeatureName` that is available as `.configSetting`.

### PR DESCRIPTION
**Pull request by matanlurey**
... for consistency with the rest of the file/features.

### CODE REVIEW

File: packages/flutter_tools/lib/src/features.dart [hunk 1]
```diff
@@ -195,6 +194,8 @@ const Feature swiftPackageManager = Feature(
   ),
 );
 
+///
+
```
Comment (andrewkolos) [inline packages/flutter_tools/lib/src/features.dart:198]: I think this was an accident

**Review by andrewkolos — CHANGES_REQUESTED** [review]
LGTM, but there is accidental formatting

File: packages/flutter_tools/lib/src/features.dart (hunk 1 above)
Comment (matanlurey) [reply packages/flutter_tools/lib/src/features.dart]: Yeah it was, fixed.

**Review by andrewkolos — APPROVED** [review]
LGTM. I don't have a strong opinion on whether config items or cli args should or shouldn't be coded into named constants, but consistency is good.

**Pull request merged**